helm/defectdojo/templates/django-deployment.yaml

```diff
@@ -84,15 +84,11 @@ spec:
       {{- end }}
       {{- if .Values.dbMigrationChecker.enabled }}
```

Comment:
```suggestion
      {{- if or .Values.dbMigrationChecker.enabled .Values.cloudsql.enabled }}
```

Reply:
Nice we had the same conclusion, I just hadn't seen the suggestion here otherwise I would've implemented it instead of creating my commit, but same exact line used